source/Sashimi.Tests/AzurePowerShellActionHandlerFixture.cs

```diff
@@ -45,7 +45,51 @@ public void ExecuteAnInlinePowerShellScript()
                                                      context.Variables.Add(KnownVariables.Action.Script.ScriptBody, psScript);
                                                  })
                                     .Execute();
+        }
+
+        [Test]
```

Comment:
@geofflamrock I reckon given the tests are the same as above, how about we use `TestCase`, what do u think?

Reply:
@johnsimons it would be nice as you are right the only real difference is the edition of powershell in the variable, however we only want to run the Windows Powershell version of the test on a Windows machine, is that somehow possible with TestCase?

Reply:
Good point